src/goe/filesystem/cli_hdfs.py

```diff
@@ -85,15 +85,16 @@ def __init__(
     ):
         logger.info("Client setup: (%s, %s, %s)" % (hdfs_host, ssh_user, tmp_dir))
 
-        super(CliHdfs, self).__init__(messages, dry_run=dry_run)
+        super(CliHdfs, self).__init__(
+            messages, dry_run=dry_run, do_not_connect=do_not_connect
+        )
 
         self._db_path_suffix = db_path_suffix
         self._hdfs_data = hdfs_data
         self._hdfs_host = hdfs_host
         self._ssh_user = ssh_user
         self._tmp_dir = tmp_dir
         self._dry_run = dry_run
-        self._do_not_connect = do_not_connect
```

Comment:
Why has the `_do_not_connect` assignment from arg been removed, whereas `_dry_run` assignment from arg has been retained?

Reply:
It was because I was only adding `do_not_connect` so I did not concern myself with the other attributes. But you are right, `self._dry_run` is also set in the parent `__init__` method so I have removed it from here too.